package/.releaserc.json

```diff
@@ -2,5 +2,11 @@
   "$schema": "https://json.schemastore.org/semantic-release",
   "branches": [
     "main"
+  ],
+  "plugins": [
+    "@semantic-release/commit-analyzer",
+    "@semantic-release/release-notes-generator",
+    "semantic-release-yarn",
+    "@semantic-release/github"
```

Comment:
Hey Jose, I think that for these plugins to work we need to install them as well.

Reply:
No, we only need to install `semantic-release-yarn`, the other three plugins starting with `@semantic-release/*` come out of the box in the `semantic-release` dependency 🙂 

https://semantic-release.gitbook.io/semantic-release/extending/plugins-list